src/test/java/org/openstreetmap/atlas/geography/atlas/items/complex/roundabout/ComplexRoundaboutTestRule.java

```diff
@@ -0,0 +1,501 @@
+package org.openstreetmap.atlas.geography.atlas.items.complex.roundabout;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.utilities.testing.CoreTestRule;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas;
+
+/**
+ * Unit test rule for {@link ComplexRoundaboutTest}.
+ *
+ * @author bbreithaupt
+ * @author savannahostrowski
+ */
+public class ComplexRoundaboutTestRule extends CoreTestRule
+{
+    private static final String CLOCKWISE_1 = "38.905336130818505,-77.03197002410889";
+    private static final String CLOCKWISE_2 = "38.90558660084624,-77.03158378601074";
+    private static final String CLOCKWISE_3 = "38.905995699991145,-77.0318305492401";
+    private static final String CLOCKWISE_4 = "38.90582872103308,-77.03239917755127";
+    private static final String CLOCKWISE_5 = "38.905494761938684,-77.03236699104309";
+
+    private static final String COUNTER_CLOCKWISE_1 = "38.905361177861046,-77.03205585479736";
+    private static final String COUNTER_CLOCKWISE_2 = "38.905528157918816,-77.03158378601074";
+    private static final String COUNTER_CLOCKWISE_3 = "38.905937257400495,-77.031809091568";
+    private static final String COUNTER_CLOCKWISE_4 = "38.90588716371307,-77.03230261802673";
+    private static final String COUNTER_CLOCKWISE_5 = "38.90551980892527,-77.03236699104309";
+
+    // Clockwise roundabout, left driving country
+    @TestAtlas(
```

Comment:
When `@TestAtlas`es become complex like that it might be easier to make them out of a JOSM layer using `loadFromJosmOsmResource` :) But this works too.